lib/spree/adyen/form.rb

```diff
@@ -132,7 +132,8 @@ def order_params order
             merchant_reference: order.number.to_s,
             country_code: order.billing_address.country.iso,
             payment_amount: (order.total * 100).to_int,
-            shopper_locale: I18n.locale.to_s.gsub("-", "_")
+            shopper_locale: I18n.locale.to_s.gsub("-", "_"),
+            shopper_email: order.email
```

Comment:
The shopper email is actually only a required field for recurring payments:
https://docs.adyen.com/developers/hpp-manual#hpppaymentfields
With the SHA1 signing method it needs to be included when calculating the signature only if it is present.